gen/definitions/ospf_vrf.yaml

```diff
@@ -72,6 +82,37 @@ attributes:
     type: String
     description: 'Controls. Choices: `unspecified`, `bfd`, `name-lookup`, `default-passive`, `segrt`. Can be an empty string. Allowed formats:\n  - Single value. Example: `bfd`\n  - Multiple values (comma-separated). Example: `bfd,default-passive`. In this case values must be in alphabetical order.'
     example: 'bfd,default-passive'
+child_classes:
+  - class_name: ospfMaxMetricLsaP
```

Comment:
Are we not able to configure the `ospfMaxMetricLsaP` class individually? In general, we should avoid using `child_classes` unless there is no other choice.